configure

```diff
@@ -6716,6 +6716,9 @@ then
 	    else
 		OPT="-g $WRAP -O3 -Wall"
 	    fi
+	    # bpo-30345: When using PGO+LTO and compile without LDFLAGS=-g,
+	    # gdb fails to get function arguments.
+	    LDFLAGS="$LDFLAGS -g"
```

Comment:
Shouldn't it be added only in debug mode? I afraid that adding `-g` in release mode can disable some optimizations.

Reply:
We already *always* pass -g to CFLAGS, even in release mode.

Extract of gcc manual page:

```
       -g  Produce debugging information in the operating system's native
           format (stabs, COFF, XCOFF, or DWARF).  GDB can work with this
           debugging information.

           On most systems that use stabs format, -g enables use of extra
           debugging information that only GDB can use; this extra information
           makes debugging work better in GDB but probably makes other
           debuggers crash or refuse to read the program.  If you want to
           control for certain whether to generate the extra information, use
           -gstabs+, -gstabs, -gxcoff+, -gxcoff, or -gvms (see below).
```

It's unrelated to optimizations. It's just an option to ask to include also debug symbols in the produced binary file (like ELF file on Linux).

If you want to get ride of them, just run "strip python". Linux vendors are able to distribute these debug symbols in a different package, like python-debuginfo on Fedora.

I'm not 100% sure, that's why I'm asking for a review :-)

Reply:
Thank you, now I got this.